contracts/KingAutomaton.sol

```diff
@@ -31,6 +33,10 @@ contract KingAutomaton is KingOfTheHill {
     proposalsData.contestPercentage = contest_pct;
     proposalsData.treasuryLimitPercentage = treasury_limit_pct;
     proposalsData.ballotBoxIDs = 99;  // Ensure special addresses are not already used
+    proposalsInitialPeriod = 7 * one_day_in_seconds;  // 7 days;
```

Comment:
Instead of 7 you should have a parameter in the constructor for both initial period and contest period.